primitives/core/Cargo.toml

```diff
@@ -32,7 +32,7 @@ substrate-bip39 = { version = "0.4.4", optional = true }
 tiny-bip39 = { version = "0.8.2", optional = true }
 regex = { version = "1.5.4", optional = true }
 num-traits = { version = "0.2.8", default-features = false }
-zeroize = { version = "1.5.4", default-features = false }
+zeroize = { version = "1.4.3", default-features = false }
```

Comment:
I read your other comment, but why do we need to downgrade here as well?

Reply:
```
error: failed to select a version for zeroize.
    ... required by package chacha20poly1305 v0.9.0
    ... which satisfies dependency chacha20poly1305 = "^0.9" of package snow v0.9.0
    ... which satisfies dependency snow = "^0.9.0" of package libp2p-noise v0.35.0
    ... which satisfies dependency libp2p-noise = "^0.35.0" of package libp2p v0.44.0
    ... which satisfies dependency libp2p = "^0.44.0" of package sc-authority-discovery v0.10.0-dev (/home/kpp/parity/substrate/client/authority-discovery)
    ... which satisfies path dependency sc-authority-discovery (locked to 0.10.0-dev) of package node-cli v3.0.0-dev (/home/kpp/parity/substrate/bin/node/cli)
    ... which satisfies path dependency node-cli (locked to 3.0.0-dev) of package chain-spec-builder v2.0.0 (/home/kpp/parity/substrate/bin/utils/chain-spec-builder)
versions that meet the requirements >=1, <1.5 are: 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.0, 1.2.0, 1.1.1, 1.1.0, 1.0.0

all possible versions conflict with previously selected packages.

  previously selected package zeroize v1.5.4
    ... which satisfies dependency zeroize = "^1.5.4" of package sc-network v0.10.0-dev (/home/kpp/parity/substrate/client/network)
    ... which satisfies path dependency sc-network (locked to 0.10.0-dev) of package beefy-gadget v4.0.0-dev (/home/kpp/parity/substrate/client/beefy)
    ... which satisfies path dependency beefy-gadget (locked to 4.0.0-dev) of package beefy-gadget-rpc v4.0.0-dev (/home/kpp/parity/substrate/client/beefy/rpc)

failed to select a version for zeroize which could resolve this conflict
```